CHANGELOG.md

```diff
@@ -33,7 +33,7 @@ kubectl label node --overwrite=true $(kubectl get nodes -o jsonpath='{.items[*].
 * Add license info to the operator container images ([#6])
 * Specify passing `-c` to `watch` in README instructions for monitoring node status ([#27])
 * Bump [bottlerocket-sdk](https://github.com/bottlerocket-os/bottlerocket-sdk) version to v0.10.1 for building the update operator's binaries. ([#21])
-* Bump the version of the golang image to 1.14.1 to match the Go toolchain version in the [bottlerocket-sdk](https://github.com/bottlerocket-os/bottlerocket-sdk). ([#31])
+* Bump the version of the golang image to 1.16.5 to match the Go toolchain version in the [bottlerocket-sdk](https://github.com/bottlerocket-os/bottlerocket-sdk). ([#31])
```

Comment:
I don't think we should modify line in released version. You can add a Changelog for new version that we planned to release next and copy this line in that. You might also want to change the issue number.

Reply:
> Don't know anything about buildx plugin would let others review that part. I only reviewed the Go version change. Just confirming, Did you also repeated the testing after updating the Go version?

Yeah, I repeated the test and worked as expected. Also, I just repeated the test for my new changes and everything looks good. Thanks